roles/openshift_certificate_expiry/library/openshift_cert_expiry.py

```diff
@@ -100,7 +241,8 @@ def filter_paths(path_list):
     return [p for p in path_list if os.path.exists(os.path.realpath(p))]
 
 
-def load_and_handle_cert(cert_string, now, base64decode=False):
+# pylint: disable=too-many-locals,too-many-branches
```

Comment:
I won't block on this but generally needing to disable these is a sign of the function needing to be broken into multiple functions.